sql/hive/src/main/scala/org/apache/spark/sql/hive/test/TestHive.scala

```diff
@@ -297,7 +298,7 @@ private[hive] class TestHiveSparkSession(
 
   protected[hive] implicit class SqlCmd(sql: String) {
     def cmd: () => Unit = {
-      () => new TestHiveQueryExecution(sql).hiveResultString(): Unit
+      () => hiveResultString(new TestHiveQueryExecution(sql)): Unit
```

Comment:
I think we can replace this by `new TestHiveQueryExecution(sql).queryExecution.executeCollect()`

Reply:
replaced